meraki/patterns/footer-default.php

```diff
@@ -6,6 +6,81 @@
+			<h2 class="has-small-font-size" style="text-transform:uppercase;letter-spacing:1px">
+				<?php
+					echo esc_html__( 'About the Blog', 'meraki' );
+				?>
+			</h2>
+			<!-- /wp:heading -->
+
+			<!-- wp:paragraph {"style":{"typography":{"fontSize":"small"},"color":{"text":"#ffffffa8"}}} -->
+			<p class="has-text-color has-small-font-size" style="color:#ffffffa8">
+				<?php
+					echo esc_html__( 'Meraki  is a travel blog theme designed to document your adventures.', 'meraki' );
+				?>	
+			</p>
+			<!-- /wp:paragraph -->
+
+		</div>
+		<!-- /wp:column -->
+
+		<!-- wp:column -->
+		<div class="wp-block-column">
+
+			<!-- wp:heading {"style":{"typography":{"textTransform":"uppercase","letterSpacing":"1px"}},"fontSize":"small"} -->
+			<h2 class="has-small-font-size" style="text-transform:uppercase;letter-spacing:1px">
+				<?php
+					echo esc_html__( 'Popular Posts', 'meraki' );
+				?>
+			</h2>
+			<!-- /wp:heading -->
+
+			<!-- wp:latest-posts {"postsToShow":3,"excerptLength":10,"displayPostDate":true,"featuredImageAlign":"left","featuredImageSizeHeight":68} /-->
+
+		</div>
+		<!-- /wp:column -->
+
+		<!-- wp:column -->
+		<div class="wp-block-column">
+
+			<!-- wp:heading {"style":{"typography":{"textTransform":"uppercase","letterSpacing":"1px"}},"fontSize":"small"} -->
+			<h2 class="has-small-font-size" style="text-transform:uppercase;letter-spacing:1px">
+				<?php
+					echo esc_html__( 'Newsletter', 'meraki' );
+				?>
+			</h2>
+			<!-- /wp:heading -->
+
+			<!-- wp:paragraph {"style":{"typography":{"fontSize":"small"},"color":{"text":"#ffffffa8"}}} -->
+			<p class="has-text-color has-small-font-size" style="color:#ffffffa8">
+				<?php
+					echo esc_html__( 'Subscribe to our email newsletter for a weekly dose of travel inspiration.', 'meraki' );
+				?>
+			</p>
+			<!-- /wp:paragraph -->
+
+			<!-- wp:jetpack/subscriptions {"buttonOnNewLine":true,"buttonWidth":"100%","submitButtonText":"SUBSCRIBE","buttonBackgroundColor":"background","textColor":"black","fontSize":"14px","customFontSize":"14px","borderRadius":50,"borderColor":"background","spacing":10,"className":"is-style-split"} -->
+			<div class="wp-block-jetpack-subscriptions wp-block-jetpack-subscriptions__supports-newline wp-block-jetpack-subscriptions__use-newline is-style-split">[jetpack_subscription_form show_subscribers_total="false" button_on_newline="true" submit_button_text="SUBSCRIBE" custom_font_size="14px" custom_border_radius="50" custom_border_weight="1" custom_button_width="100%" custom_padding="15" custom_spacing="10" submit_button_classes="has-14-px-font-size has-background-border-color has-text-color has-black-color has-background has-background-background-color" email_field_classes="has-14-px-font-size has-background-border-color" show_only_email_and_button="true" success_message="Success! An email was just sent to confirm your subscription. Please find the email now and click 'Confirm Follow' to start subscribing."]</div>
```

Comment:
[Here](https://github.com/Automattic/themes/blob/trunk/remote/patterns/subscribe.php)'s how we handle that for Remote